tgui/packages/tgui/interfaces/PlayerPanel.js

```diff
@@ -415,6 +415,30 @@ const PunishmentActions = (props, context) => {
         </Stack>
       </Section>
 
+      <Section level={2} title="Human Name">
+        <Stack
+          align="right"
+          grow={1}
+        >
+          <Button
+            width="100%"
+            icon="clipboard-list"
+            color="average"
+            content="Human name reset"
+            disabled={!hasPermission(data, "reset_human_name")}
+            onClick={() => act("reset_human_name")}
+          />
+          <Button
+            width="100%"
+            height="100%"
+            icon="clipboard-list"
+            color="bad"
+            content="Human name ban"
+            disabled={!hasPermission(data, "ban_human_name")}
+            onClick={() => act("ban_human_name")}
```

Comment:
if they are already unbanned the button should show that by clicking it you'll unban them